roles/pulp_content/README.md

```diff
@@ -8,6 +8,9 @@ Role Variables
 
 * `pulp_content_bind`: Interface and Port where Pulp Content [`gunicorn` service will
   listen.](https://docs.gunicorn.org/en/stable/settings.html#bind)
+* `gunicorn_num_workers`: The number of Gunicorn `aiohttp.worker.GunicornWebWorker`
```

Comment:
Actually maybe this name is confusing because we use gunicorn in two places (pulpcore-content) and pulpcore-api. So perhaps we need a setting for each and have them named appropriately.